components/prize_pool/wikis/ageofempires/prize_pool_custom.lua

```diff
@@ -0,0 +1,81 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Opponent = require('Module:Opponent')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {10, 6, 4, 2}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.opponentLibrary = 'Opponent/Custom'
```

Comment:
Should we change the handling in the PPT Commons so the require is here instead of in PPT Commons?

Reply:
What would that change? Also /cc @hjpalpha

Reply:
Well this would be 
```suggestion
	args.opponentLibrary = require('Opponent/Custom')
```
And in PPT we'd change
`Opponent = require(args.opponentLibrary)` to `Opponent = args.opponentLibrary`

Reply:
Should have worded my question differently, what would be the reason behind this change?

Reply:
i am fine with both versions

Reply:
> Should have worded my question differently, what would be the reason behind this change?

I saw the implementation on the Custom side (first Custom with one) and felt I didn't like the String input :D

Reply:
Should that be done in another PR, or should we add it to this one?

Reply:
Another PR, just wanted opinions

Reply:
I don't really care. One negative thing i can think of would be when someones tries to `mw.logObject` the prize pool args, which then would be bloated with the module, right?